fuzzing/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# License: Boost 1.0
+# By Paul Dreik 2020
+
+set(INC ${SOURCES_DIR})
+message(STATUS "inc is ${INC}")
+add_executable(fuzz_TestSpecParser fuzz_TestSpecParser.cpp)
+target_link_libraries(fuzz_TestSpecParser PRIVATE Catch2::Catch2)
+target_include_directories(fuzz_TestSpecParser PRIVATE ${INC})
```

Comment:
Is this needed?

Reply:
Which part did you think of? The status message, the include, the link, fuzzing testspecparser at all?

Reply:
Fair.

The manual include setting.

Reply:
You are right, fixed it.